bounties/other/bludit/1/vulnerability.json

```diff
@@ -0,0 +1,57 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-08-12",
+    "AffectedVersionRange": "*",
+    "Summary": "Cross Site Scripting",
+    "Author": {
+        "Username": "Tibinsunny",
+        "Name": "Tibin Sunny"
+    },
+    "Package": {
+        "Registry": "other",
+        "Name": "Bludit",
+        "URL": "http://www.bludit.com/",
+        "Downloads": "0"
+    },
+    "CWEs": [
+        {
+            "ID": "79",
+            "Description": "Improper User Input Sanitization"
+        }
+    ],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "",
+        "AC": "",
+        "PR": "",
+        "UI": "",
+        "S": "",
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "3.3"
```

Comment:
I would higher a bit the CVSS score ... even if the user needs to be a `admin`, it's still possible get an impactful behavior :smile:

![Screenshot from 2020-08-12 23-24-25](https://user-images.githubusercontent.com/33063403/90069822-225e7200-dcf3-11ea-877e-ff3feff86b88.png)


Cheers,
Mik

Reply:
@Mik317 
Thanks for pointing out :-)  .I have Updated CVSS Score. 😄😄

Regards,
Tibin